src/client/components/pages/RoomAdmin/RoomAdmin.tsx

```diff
@@ -138,7 +175,40 @@ const RoomAdmin: FunctionComponent = (): ReactElement => {
   useEffect((): void => {
     void loadRooms();
   }, [loadRooms]);
-
+  /**
+   * Populates building names for the building filter dropdown.
+   */
+  const buildingDropDown = buildingNameList.map((buildingName) => (
+    { value: buildingName, label: buildingName }
+  ));
+  /**
+   * Filters rooms list based on selected campus, building, room and capacity.
```

Comment:
Small note, but I would also change "room" to "room name" here as well for clarity.